js/src/css/shared/_woocommerce-admin.scss

```
@@ -76,3 +76,8 @@
 		}
 	}
 }
+
+// FIX in WC 7.3 for adding margin in the first question label
```

WDYT of adding our compatibility code marker?
```suggestion
// compatibility-code "WC >= 7.3" -- adding margin in the first question label.
```